nutkit/protocol/requests.py

```diff
@@ -172,6 +267,31 @@ def __init__(self, driverId):
         self.driverId = driverId
 
 
+class VerifyAuthentication:
+    """
+    Request to verify authentication on the driver.
+
+    instance corresponding to the specified driver id.
+    Backend should respond with a DriverIsAuthenticated response or an Error
+    response.
+    """
+
+    def __init__(self, driver_id, auth_token):
+        self.driverId = driver_id
+        self.authToken = auth_token
```

Comment:
```suggestion
        self.authorizationToken = auth_token
```

should probably be called that to be more in line with `NewDriver` and `NewSession`. Even though I really don't' like the unnecessary verbosity of it.

Reply:
Good point, thanks for updating it. 👍